City-of-Helsinki/berth-reservations-admin
# VEN-665 | Add selected rows indicator and clear action to customer list table tools

## Description :sparkles:

Adds selected row count indicator and "clear all" action to the customer list's table tools.

### Closes :no_good_woman:

[VEN-665](https://helsinkisolutionoffice.atlassian.net/browse/VEN-665)

## Testing :alembic:

### Automated tests :gear:️

`browser-tests/customerListView.ts`

### Manual testing :construction_worker_man:

- Navigate to customers page
- Select customers with the selection column
- Try selections on different pages
- The correct number of selections should be visible in the table tools
- Clear all button should clear selections from every page
- Send message button should be disabled if there are no selections
